src/wp-includes/wp-db.php

```diff
@@ -1347,6 +1367,31 @@ public function escape_by_ref( &$string ) {
 		}
 	}
 
+	/**
+	 * Escapes an identifier for a MySQL database (e.g. table/field names).
+	 *
+	 * @since 6.0.0
+	 *
+	 * @param string $identifier Identifier to escape.
+	 */
+	public function escape_identifier( $identifier ) {
+		return '`' . $this->_escape_identifier_value( $identifier ) . '`';
+	}
+
+	/**
+	 * Escapes an identifier value, but does not add the quote marks itself.
+	 * https://dev.mysql.com/doc/refman/8.0/en/identifiers.html
+	 *   Permitted characters in quoted identifiers include the full Unicode Basic Multilingual Plane (BMP), except U+0000
+	 *   To quote the identifier itself, then you need to double the character, e.g. `a``b`
+	 *
+	 * @since 6.0.0
+	 *
+	 * @param string $identifier Identifier to escape.
+	 */
```

Comment:
```suggestion
	/**
	 * Escapes an identifier value.
	 *
	 * Escapes an identifier value without adding the surrounding quotes. 
	 * 
	 * - Permitted characters in quoted identifiers include the full Unicode Basic Multilingual Plane (BMP), except U+0000
	 * - To quote the identifier itself, then you need to double the character, e.g. `a``b`
	 *
	 * @link https://dev.mysql.com/doc/refman/8.0/en/identifiers.html
	 * @since 6.0.0
	 *
	 * @param string $identifier Identifier to escape.
	 */
```

I'm not sure if private functions need `@private` as it's the case by definition.

Reply:
You're right about '@private', but checking other PHP methods, the [PHPDoc Tags](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#phpdoc-tags) use `@access private` and follows the `@since` tag (I've added this locally for now, but have noticed the unit tests have started to fail, so will upload in a bit).

As an aside, I have seen `@private` used in JavaScript quite a bit.